app/code/core/Mage/Core/Model/Layout/Update.php

```diff
@@ -405,10 +405,16 @@ protected function _getUpdateString($handle)
     public function fetchRecursiveUpdates($updateXml)
     {
         foreach ($updateXml->children() as $child) {
-            if (strtolower($child->getName()) == 'update' && isset($child['handle'])) {
-                $this->merge((string)$child['handle']);
-                // Adding merged layout handle to the list of applied hanles
-                $this->addHandle((string)$child['handle']);
+            if ((strtolower($child->getName()) == 'update') && isset($child['handle'])) {
+                $allow = true;
+                if (isset($child['ifconfig']) && ($configPath = (string)$child['ifconfig'])) {
+                    $allow = Mage::getStoreConfigFlag($configPath);
+                }
+                if ($allow) {
```

Comment:
```suggestion
            if (strtolower($child->getName()) == 'update' && isset($child['handle'])) {
                $allow = true;
                if (isset($child['ifconfig'])) {
                    $allow = Mage::getStoreConfigFlag((string)$child['ifconfig']);
                }
                if ($allow) {
```

Reply:
I copy pasted logic from:

https://github.com/OpenMage/magento-lts/blob/412d44f42036ded84f5d7e96312605de3bdc80f0/app/code/core/Mage/Core/Model/Layout.php#L290-L295

Reply:
then we reuse it :+1: